latrine_django/latrineapp/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
 
-# Create your models here.
+
+class Place(models.Model):
+    """Contains basic information about each facility displayed on the map."""
+
+    organization = models.TextField(max_length=255, default='', null=True)
+    website = models.TextField(max_length=255, default='', null=True)
+    short_description = models.TextField(max_length=1000, default='', null=True)
+    address = models.TextField(max_length=255, default='', null=False)  # required
+    phone = models.TextField(max_length=255, default='', null=True)
+    lat = models.FloatField(null=False)  # required
+    lng = models.FloatField(null=False)  # required
+    gender = models.TextField(max_length=255, default='', null=True)
```

Comment:
i wonder how much performance matters with field length. e.g. the 'gender' field probably doesn't need to allow for 255 characters but what would shortening that save us? and maybe there's a security concern with fields that are longer than they need to be? same with the 'phone' field. what do you think about shortening those two?

Reply:
I'm not sure about the performance, but I'll definitely make them shorter for security reasons.